README.md

```diff
@@ -64,10 +64,17 @@ To execute the Terraform script:
 
 ```
 module "iam-config" {
-  source     = "github.com/uptycslabs/terraform-azure-iam-config"
+  source     = "github.com/uptycslabs/terraform-azurearm-ad-config"
```

Comment:
github.com/uptycslabs/terraform-azurerm-ad-config

Reply:
Done

Reply:
@diyor-uptycs  there is typo. Can you fixt it